code/datums/design/designs/tools/general.dm

```diff
@@ -76,6 +79,12 @@
 	material_costs = list(
 		"tip" = 250
 	)
+	material_constraints = list(
```

Comment:
question; if we added material_constraints to the obj level, why not autodetect it from that?

Reply:
did i add it to the obj level. it's purely a design thing rn but ig crafting caan use it??